### ISSUE TITLE
WZA One kurier

### ISSUE THREAD

**Issue by d3stozaur** [allegro/allegro-api#5929]
Dzień dobry
przy próbie zamówienia przesyłki One kurier 
   "serviceId": 17466151
dostajemy odpowiedź z API:
"Usługa niedostępna u tego przewoźnika"

Czy może to byś spowodowane błędnym requestem? Czy można jakoś sprawdzić które usługi są dostępne dla danego konta allegro?

**Comment by PrzemyslawLukanowski** [comment]
Tak, serviceId jest unikalne dla każdego użytkownika i sprawdzisz je za pomocą [GET /parcel-management/delivery-services](https://developer.allegro.pl/documentation/#operation/getAvailableDeliveryServices). W sekcji `options` znajdziesz dostępne opcje dodatkowe dla danej usługi dostawy.

Poniżej znajdziesz również pełne rozróżnienie metod dostawy na serwis:

- Metoda dostawy: Allegro One Punkt (UPS)
   Serwis: Allegro One (UPS)
- Metoda dostawy: Allegro One Box (UPS)
   Serwis: Allegro One (UPS)
- Metoda dostawy: Allegro One Punkt (One Kurier)
   Serwis: Allegro One Punkt (One Kurier) (na Sandbox: Allegro One Punkt (One Kurier) - dostawa jutro)
- Metoda dostawy: Allegro One Box (One Kurier)
   Serwis:Allegro One Box (One Kurier) (na Sandbox: Allegro One Box (One Kurier) - dostawa jutro)
- Metoda dostawy: Allegro One Kurier - dostawa dzisiaj
   Serwis: Allegro One Kurier
- Metoda dostawy: Allegro One Kurier - dostawa jutro
   Serwis: Allegro One Kurier
- Metody z dopiskiem (do 01.07.2022) oraz pobraniowe NIE SĄ obsługiwane w WzA.

Jeśli problem nadal będzie występować, podaj proszę commandID, za pomocą którego próbujesz utworzyć paczkę.

**Comment by d3stozaur** [comment]
Niestety problem występuje a wg api dana metoda dostawy jest dostępna.
commandID 1628d82e-7dc4-4000-becb-1e5e5703e12a

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję. Błąd występuje, ponieważ próbujesz nadać przesyłkę z miejscowości, której kod pocztowy pocztowy nie jest obsługiwany w strefie One Kuriera. Dostępność usługi Allegro One Kurier sprawdzisz [na tej stronie](https://allegro.pl/kampania/one/kurier#gdzie).

**Comment by kamilkomtek** [comment]
Witam
1. W takim razie proszę podać jakie kody z Krakowa działają?
Bo na tej stronie jest też informacja że ww metody przesyłek działają w aglomeracji krakowskiej a Wasz pracownik uruchomił mi taką metodę wysyłki i specjalnie weryfikował kod pocztowy.

2. Dlaczego dla danego kodu można nadać etykietę przez portal allegro ręcznie i wtedy kod jest poprawny?
A nie można zrobić tego po API?

3. Dlaczego na chat allegro wszyscy twierdzą że na pewno działa, a po miesiącu okazuje się że jednak nie działa?

**Comment by PrzemyslawLukanowski** [comment]
Wracam z odpowiedzią po ustaleniach z zespołem odpowiedzialnym za usługę. Kod pocztowy jest prawidłowy - przepraszam za pomyłkę, ale wygląda na to, że wczoraj kod mógł być chwilowo nieaktywny na stronie. Przesyłka, którą próbujecie nadać, to przesyłka w ramach dostawy tego samego dnia. Paczki tego typu powinny być nadane do 10:30, ponieważ jest to graniczna godzina nadania. Możecie zatem powtórzyć request - do wspomnianej godziny powinien zakończyć się prawidłową odpowiedzią.

**Comment by kamilkomtek** [comment]
Witam
Dzisiaj nadal występuje ten sam błąd. Nie można nadać przesyłki po API. Wyskakuje komunikat: Error: Usługa niedostępna u tego przewoźnika

**Comment by kamilkomtek** [comment]
Oczywiście sprawdzałem przed godziną graniczną.

**Comment by PrzemyslawLukanowski** [comment]
Spróbujcie proszę jeszcze usunąć wartość w `pickup.pointId`, przekażcie tam null - Allegro One Kurier nie daje możliwości odbioru przesyłki z punktu.

**Comment by kamilkomtek** [comment]
Witam
A dzisiaj znowu błąd Error: Wystąpił błąd komunikacji z API przewoźnika

**Comment by MaciejFrackowiak** [comment]
Proszę podaj identyfikator komendy, dla której pojawił się komunikat.

**Comment by kamilkomtek** [comment]
WitamSprawdzalismy o roznych godzinach i czas nadania nie ma tutaj znaczenia. Zarowno przed godzina graniczna jak i po caly czas wystepuje ten sam blad.26 lip 2022 09:23 PrzemyslawLukanowski ***@***.***> napisał(a):

Wracam z odpowiedzią po ustaleniach z zespołem odpowiedzialnym za usługę. Kod pocztowy jest prawidłowy - przepraszam za pomyłkę, ale wygląda na to, że wczoraj kod mógł być chwilowo nieaktywny na stronie. Przesyłka, którą próbujecie nadać, to przesyłka w ramach dostawy tego samego dnia. Paczki tego typu powinny być nadane do 10:30, ponieważ jest to graniczna godzina nadania. Możecie zatem powtórzyć request - do wspomnianej godziny powinien zakończyć się prawidłową odpowiedzią.

**Comment by MaciejFrackowiak** [comment]
@kamilkomtek Proszę podaj nam przykładowy identyfikator komendy, ponieważ tylko tak możemy zweryfikować dany przypadek.
